.env.example

```diff
@@ -1,14 +1,15 @@
+BRIZO_URL=http://localhost:8080
+
 BRIZO_MYSQL_USER=root
 BRIZO_MYSQL_PASSWORD=secret
 BRIZO_MYSQL_HOST=127.0.0.1
 BRIZO_MYSQL_PORT=33062
 BRIZO_MYSQL_NAME=brizo
 
-OAUTH_GITHUB_CLIENT_ID=exampleClientId
-OAUTH_GITHUB_CLIENT_SECRET=exampleClientSecret
+OAUTH_GITHUB_CLIENT_ID=3107e9dc652f2ffb95a7
+OAUTH_GITHUB_CLIENT_SECRET=cddd23dc61e6faa0b1d28233d9e9b958272757ad
```

Comment:
Intentional. I setup a `brizo-dev` application for gentux for dev simplicity. Probably bad practice for a public repo but figured we could change it easily if it becomes an issue.